flutter-studio/src/io/flutter/editor/ColorPickerAndroidStudio.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2019 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.editor;
+
+import com.android.tools.idea.ui.resourcechooser.colorpicker2.ColorPickerBuilder;
+import com.intellij.openapi.ui.popup.Balloon;
+import com.intellij.openapi.ui.popup.BalloonBuilder;
+import com.intellij.openapi.ui.popup.JBPopupFactory;
+import com.intellij.ui.awt.RelativePoint;
+import io.flutter.editor.GenericColorPicker;
+import kotlin.Unit;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.KeyEvent;
+
+import java.awt.*;
+import javax.swing.*;
+
+public class ColorPickerAndroidStudio implements GenericColorPicker {
```

Comment:
Chatted about using a new extension point to create different color picker instances depending on whether the flutter-studio module was loaded.

Reply:
done